homeassistant/components/media_player/__init__.py

```diff
@@ -214,6 +213,25 @@ class MediaPlayerDeviceClass(StrEnum):
     ATTR_MEDIA_REPEAT,
 ]
 
+# mypy: disallow-any-generics
+
+
+class CacheImage(TypedDict, total=False):
+    """Class to hold a cached image."""
+
+    lock: asyncio.Lock
+    content: tuple[bytes | None, str | None]
+
+
+class ImageCache(TypedDict, total=True):
```

Comment:
```suggestion
class ImageCache(TypedDict):
```
`total=True` is the default.